### DISCUSSION TITLE
48E13896-CBCD-ABE0-464A-6ACE9C2C16F0

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#132] [General]
# 48E13896-CBCD-ABE0-464A-6ACE9C2C16F0

Kepler-1654 is a Sun-like star about 1,884 light-years away in the constellation Lyra. It has a single Saturn-sized planet that orbits within the star's habitable zone

https://celestia.mobi/addon?item=48E13896-CBCD-ABE0-464A-6ACE9C2C16F0

**Comment by exocritic5457** [comment]
I think.... you forgot an atmosphere?

**Reply by DaveBowman2001** [reply]
The original addon, when it was released to the Forums, had no atmosphere defined, but the latest update (231110) should have this added now
